### PR TITLE
changes: consolidate action names

### LINKED ISSUE #9824: Inconsistent history event names
### Describe the issue

The name of history events are inconsistent, see [this section](https://github.com/WeblateOrg/weblate/blob/main/weblate/trans/models/change.py#L284-L422). Some examples:

- Resource update: noun + verb
- Translation completed: noun + verb past
- New translation: no verb
- Removed comment: verb past + noun

They should be consistent like "Something happened" (noun + verb past):

- Resource updated
- Translation completed
- Translation added 
- Comment removed

### I already tried

- [X] I've read and searched [the documentation](https://docs.weblate.org/).
- [X] I've searched for similar issues in this repository.

### Steps to reproduce the behavior

Open [this section](https://github.com/WeblateOrg/weblate/blob/main/weblate/trans/models/change.py#L284-L422)

or

Open `/changes/browse/PROJECT/COMPONENT/` to see all events in the GUI.

### Expected behavior

_No response_

### Screenshots

_No response_

### Exception traceback

_No response_

### How do you run Weblate?

weblate.org service

### Weblate versions

_No response_

### Weblate deploy checks

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by nijel**
## Proposed changes

Consistently use noun + verb past.

Fixes #9824

## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added documentation to describe my feature.
- [ ] I have squashed my commits into logic units.
- [ ] I have described the changes in the commit messages.

### CODE REVIEW

**Comment by urbalazs** [comment]
I prefer these changes:

- Repository was reset -> Repository reset
- Repository was merged -> Repository merged
- Repository was rebase -> Repository rebased
- New string added -> String added
- New language added -> Language added
- New language requested -> Language requested
- Trasnlation replaced file by upload -> Translation replaced by file upload

I am not sure about the last one. Please note there is a typo in Tras**n**lation

**Comment by nijel** [comment]
@urbalazs Thanks for the feedback, I've applied your suggestions.

File: weblate/trans/models/change.py [hunk 1]
```diff
@@ -296,109 +296,109 @@ class Change(models.Model, UserDisplayMixin):
...
+        (ACTION_RENAME_COMPONENT, gettext_lazy("Component renamed")),
         # Translators: Name of event in the history
-        (ACTION_MOVE_COMPONENT, gettext_lazy("Moved component")),
+        (ACTION_MOVE_COMPONENT, gettext_lazy("Component moved")),
         # Using pgettext to differentiate from the plural
         # Translators: Name of event in the history
         (
             ACTION_NEW_STRING,
-            pgettext_lazy("Name of event in the history", "New string to translate"),
+            pgettext_lazy("Name of event in the history", "String added"),
```
Comment (Geeyun-JY3) [inline weblate/trans/models/change.py:380]: There are two `String added` after changing `New string added` to `String added`. It is needed to distinguish these two events.

Comment (nijel) [reply weblate/trans/models/change.py:380]: Good point, I think it's time to drop this event. Tracked separately at https://github.com/WeblateOrg/weblate/issues/10153

**Pull request merged**